sui/deploy-contract.js

```diff
@@ -272,7 +272,7 @@ async function upgrade(keypair, client, supportedPackage, policy, config, chain,
 async function mainProcessor(args, options, processor) {
     const config = loadConfig(options.env);
     const [keypair, client] = getWallet(config.sui, options);
-    await printWalletInfo(keypair, client, config.sui, options);
+    await printWalletInfo(keypair.toSuiAddress(), client, config.sui, options);
```

Comment:
do this within `printWalletInfo`

Reply:
ah, check if it has the sui keypair type internally and only then convert it (or check if it's a string and don't convert)

Reply:
done